examples/nek5000/ex1.usr

```diff
@@ -871,7 +885,7 @@ c-----------------------------------------------------------------------
       integer ceed,err
 
 c     Init ceed library
-      call ceedinit('/cpu/self'//char(0),ceed,err)
+      call ceedinit('/gpu/occa'//char(0),ceed,err)
```

Comment:
How can we make this a run-time option? (See also the email discussion with @tzanio.) It should be possible to build and run this example even without a CUDA device, but here I needed to edit the code to confirm that it works.